server/bin/pbench-index

```diff
@@ -128,13 +129,24 @@ def main(options, name):
         idxctx.templates.dump_templates()
         return 0
 
-    idxctx.logger.debug("{}.{}: starting", name, idxctx.TS)
+    idxctx.logger.info("{}.{}: starting", name, idxctx.TS)
```

Comment:
Are you sure you want to upgrade this from `debug()` to `info()`?

Analogously, lines 146 and 149 and possibly 139 and 142.